### DISCUSSION TITLE
v4 버젼에서 Carousel 타입의 메세지를 보내는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by dlehdanakf** [storycraft/node-kakao#253] [Q&A] [answered]
예시에 이미지와 텍스트만 있던데, 해당 타입의 메세지들만 보낼 수 있는지 궁금해서 여쭙습니다~ :)

**Comment by storycraft** [comment, marked as answer]
메세지의 경우 전송 가능한 모든 타입이 전송가능합니다. 일부 메세지는 타입에 맞는 attachment 객체가 필요합니다. Carousel 은 카카오링크 메세지 형식중 하나 입니다. https://github.com/storycraft/node-kakao/blob/51960f48f23d05134b748cf873f09fdbc7362831/src/chat/chat-type.ts#L40
다만 현재 카카오링크 메세지의 전송이 제한되어 있어 #176  Carousel 형식의 카카오링크 메세지 전송은 어려울것 같습니다.
